packages/components/carousel/src/carousel.vue

```diff
@@ -13,6 +13,7 @@
           "
           type="button"
           :class="[ns.e('arrow'), ns.em('arrow', 'left')]"
+          aria-label="Carousel Arrow Left"
```

Comment:
IMO, This writing style is not conducive to internationalization, as screen readers can only prompt by English. 

Perhaps adding it to `locale` folder would be better. refer #11817

Reply:
Done. Not entirely certain about the naming though.